functions/postSurveyComplete.protected.ts

```diff
@@ -45,13 +45,17 @@ const saveSurveyInInsights = async (
   postSurveyConfigJson: OneToManyConfigSpec[],
   memory: BotMemory,
   surveyTask: TaskInstance,
+  surveyTaskAttributes: any,
 ) => {
   const handlerPath = Runtime.getFunctions()['helpers/insightsService'].path;
   const buildSurveyInsightsData = require(handlerPath)
     .buildSurveyInsightsData as BuildSurveyInsightsData;
 
-  const taskAttributes = JSON.parse(surveyTask.attributes);
-  const finalAttributes = buildSurveyInsightsData(postSurveyConfigJson, taskAttributes, memory);
+  const finalAttributes = buildSurveyInsightsData(
+    postSurveyConfigJson,
+    surveyTaskAttributes,
+    memory,
+  );
   console.log('finalAttributes: ', JSON.stringify(finalAttributes));
```

Comment:
I know this isn't part of this PR -- but will this `console.log` possibly trigger pages?

Reply:
No, `console.log`s do not trigger the reporter, only `warn`s and `error`s. But I will remove this line anyways.